### PR TITLE
Added a penguin plushie!

### PR DESCRIPTION
**Pull request by CMDR-Piboy314**
## About the PR

Added a new penguin plushie entity, and put it in the space villain arcade and the plushie crate.

## Why / Balance

It's cute!

## Technical details

No C# changes in this PR.

## Media

Old sprite (shows functionality)
https://github.com/space-wizards/space-station-14/assets/92357316/f4c24d6c-8100-47e4-95ee-78e21f922b66

New sprite
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

None.

**Changelog**

:cl: Piboy314, netwy
- add: Added a penguin plushie!

### CODE REVIEW

**Comment by CatMagic1** [comment]
powercreep

**Comment by UbaserB** [comment]
oh my god it looks so dumb i love it

**Comment by potato1234x** [comment]
linux penguin?

**Comment by asperger-sind** [comment]
> linux penguin?

the description "i use arch btw!"implies that

**Comment by deltanedas** [comment]
sorry void better

**Comment by JoeHammad1844** [comment]
traitor grenade penguin plushie :trollface:

**Comment by lzk228** [comment]
plushie_linux-export
plushie_linux_02-export
here, you can use one of those
(by netwy)

**Comment by CMDR-Piboy314** [comment]
Epic! I'll switch out the sprite and edit the :cl: in a bit!

**Comment by CMDR-Piboy314** [comment]
Sprite switched out, netwy added to :cl:, tested, checks... mostly successful- the error seems unrelated to my PR, as it's ``System.InvalidOperationException : Called Delete on an entity already being deleted. Entity: HandcuffsDummy (2948, HandcuffsDummy)``, no conflicts. Ready for merge!

File: Resources/Textures/Objects/Fun/toys.rsi/meta.json
Comment (lzk228) [inline Resources/Textures/Objects/Fun/toys.rsi/meta.json:1]: you also should update copyright

**Review by lzk228 — CHANGES_REQUESTED** [review]

File: Resources/Textures/Objects/Fun/toys.rsi/meta.json
Comment (CMDR-Piboy314) [reply Resources/Textures/Objects/Fun/toys.rsi/meta.json:1]: done

**Comment by CMDR-Piboy314** [comment]
Anything more I need to do to get this merged?

**Comment by lzk228** [comment]
> Anything more I need to do to get this merged?

patience. a lot of it

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**